shared/config/modules.php

```diff
@@ -1,13 +1,12 @@
 <?php
 
 return [
-    /**
-     * ---------------------------------------------------------
-     * Project modules
-     * ---------------------------------------------------------
-     *
-     * Project modules which framework will load if enabled
-     */
-    'modules' => [
-    ]
+    'Web' => [
```

Comment:
The modules array should initially be empty and will be updated by commands like demo or other